Makefile

```diff
@@ -117,26 +118,33 @@ codegen:
 	$(CODEGEN) all $(PKG)/pkg/package-server/client $(PKG)/pkg/package-server/apis "packagemanifest:v1alpha1"
 
 container-codegen:
-	docker build -t olm:codegen -f codegen.Dockerfile .
+	docker build -t olm:codegen -f mockgen.Dockerfile .
```

Comment:
shouldn't this still be `codegen.Dockerfile`?

Reply:
yup